aws-iot-scheduledaudit/aws-iot-scheduledaudit.json

```diff
@@ -0,0 +1,134 @@
+{
+  "typeName": "AWS::IoT::ScheduledAudit",
+  "description": "Scheduled audits can be used to specify the checks you want to perform during an audit and how often the audit should be run.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-iot.git",
+  "definitions": {
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
+      "properties": {
+        "Key": {
```

Comment:
Should we add a regex pattern here so as to ensure customer doesn't try to add aws prefix tags?

Reply:
I think we are leaning towards having the service throw a well formatted exception instead of having the handler throw it. It's a conscious choice we have made throughout to keep validations to minimum and let the service handle it.